Tests/ClientTest.php

```diff
@@ -3,97 +3,128 @@
 namespace TruckersMP\Tests\API;
 
 use Carbon\Carbon;
-use TruckersMP\API\APIClient;
-use TruckersMP\Types\Ban;
-use TruckersMP\Types\Bans;
-use TruckersMP\Types\Player;
-
-class ClientTest extends \PHPUnit_Framework_TestCase
+use PHPUnit\Framework\TestCase;
+use TruckersMP\Client;
+use TruckersMP\Models\BanModel;
+use TruckersMP\Models\BansModel;
+use TruckersMP\Models\GameTimeModel;
+use TruckersMP\Models\PlayerModel;
+use TruckersMP\Models\RulesModel;
+
+class ClientTest extends TestCase
 {
-    private $testAccount = 585204;
+    const TEST_ACCOUNT = 585204;
```

Comment:
We should add a proper test account for production (maybe even several accounts for checking private states)